server/src/test/java/org/elasticsearch/nodesinfo/NodeInfoStreamingTests.java

```diff
@@ -163,7 +163,7 @@ private static NodeInfo createNodeInfo() {
                         randomAlphaOfLengthBetween(3, 10),
                         randomAlphaOfLengthBetween(3, 10),
                         randomAlphaOfLengthBetween(3, 10),
-                        VersionUtils.randomVersion(random()),
+                        randomAlphaOfLengthBetween(3, 10),
```

Comment:
Just a note for the record: I am using `randomAlphaOfLengthBetween(6, 32)` as a replacement for version (to cover "8.12.0" as well as a build hash). But it's just a note: I think every random "any string" have to work.